<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegundaTela</class>
 <widget class="QMainWindow" name="SegundaTela">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SegundaTela</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.784, y1:0.853, x2:0.017, y2:0.0113636, stop:0.113636 rgba(0, 210, 165, 255), stop:1 rgba(0, 118, 118, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbTitulo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:white;
font: 15pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:white;
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionRegister"/>
    <addaction name="actionLogin"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionRegister">
   <property name="text">
    <string>Register</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
